src/main/java/org/alliancegenome/curation_api/jobs/executors/AgmDiseaseAnnotationExecutor.java

```diff
@@ -45,6 +47,11 @@ public void runLoad(BulkLoadFile bulkLoadFile) {
 
 			IngestDTO ingestDto = mapper.readValue(new GZIPInputStream(new FileInputStream(bulkLoadFile.getLocalFilePath())), IngestDTO.class);
 			bulkLoadFile.setLinkMLSchemaVersion(getVersionNumber(ingestDto.getLinkMLVersion()));
+			if (!validSchemaVersion(bulkLoadFile.getLinkMLSchemaVersion(), AGMDiseaseAnnotationDTO.class)) {
+				bulkLoadFile.setBulkloadStatus(JobStatus.FAILED);
+				bulkLoadFileDAO.merge(bulkLoadFile);
+				return;
+			}
```

Comment:
@markquintontulloch I would suggest moving more of this logic block into the `validateSchemaVersion` method 
```java
if(!validateSchemaMethod(bulkLoadFile, ingestDto, AGMDiseaseAnnotationDTO.class) return;
```

Reply:
Done